ci_feature_prod/main.sh

```diff
@@ -91,8 +91,8 @@ if [  -e "/etc/config/settings/config-version" ] && [  -s "/etc/config/settings/
 fi
 
 # Check for internet connectivity
-RET=`curl -s -o /dev/null -w "%{http_code}" http://www.microsoft.com/`
-if [ $RET -eq 200 ]; then
+RET=`curl -s -o /dev/null -w "%{http_code}" https://abc.blob.core.windows.net`
```

Comment:
hi rashmi - This is also different for gov clouds, so are we curl-ing public cloud always ?

Reply:
Hi Vishwa, I havent fixed that bug yet. I will be working on that fix next. May be I will do it in the same branch and merge it once and for all.

Reply:
have included the domain check changes